app/editor/src/features/admin/sources/constants/defaultSource.ts

```diff
@@ -14,6 +14,7 @@ export const defaultSource: ISourceForm = {
   useInTopics: false,
   ownerId: '',
   productId: '',
+  productSearchGroupId: '',
```

Comment:
is it possible that a source might actually be associated with multiple Product types?  I thought I heard Scott or maybe Carolynn mention something like that?